packages/story-editor/src/components/panels/design/textStyle/karma/textStyle.other.karma.js

```diff
@@ -86,9 +86,14 @@ describe('Text Style Panel', () => {
 
       // Select first text as well (the second is selected by default).
       await fixture.events.keyboard.down('Shift');
-      await fixture.events.click(
-        fixture.editor.canvas.framesLayer.frames[1].node
+      await fixture.events.mouse.clickOn(
+        fixture.editor.canvas.framesLayer.frames[1].node,
+        10,
+        10
       );
+      // await fixture.events.click(
+      //   fixture.editor.canvas.framesLayer.frames[1].node
+      // );
```

Comment:
oops. I'll delete this. old remnant. my bad

Reply:
@littlemilkstudio friendly ping :) before we merge this by accident